### PR TITLE
Refactor: improve debugging experience

### PR DESCRIPTION
**Pull request by ZhengLin-Li**
Often, I do not know which sentence belongs to which part when debugging, which is very annoying. 

This PR separates them out by adding an additional `\n` return.

### CODE REVIEW

**Comment by mrigger** [comment]
Could you show a before/after output? I noticed that the logging no longer works correctly, as newlines are currently printed as `\n`, rather than an actual linebreak.

File: src/sqlancer/ComparatorHelper.java [hunk 1]
```diff
@@ -84,35 +84,42 @@ public static List<String> getResultSetFirstColumnAsString(String queryString, E
...
+                    firstResultSet.size(), secondResultSet.size(), firstQueryString, secondQueryString);
             throw new AssertionError(assertionMessage);
         }
 
-        Set<String> firstHashSet = new HashSet<>(resultSet);
+        Set<String> firstHashSet = new HashSet<>(firstResultSet);
         Set<String> secondHashSet = new HashSet<>(secondResultSet);
 
         if (!firstHashSet.equals(secondHashSet)) {
             Set<String> firstResultSetMisses = new HashSet<>(firstHashSet);
```
Comment (ZhengLin-Li) [inline src/sqlancer/ComparatorHelper.java:109]: here, what firstResultSetMisses should be (secondHashSet-firstHashSet)?, e.g. 
```java
Set<String> firstResultSetMisses = new HashSet<>(secondHashSet);
firstResultSetMisses.removeAll(firstHashSet);
```

Comment (bajinsheng) [reply src/sqlancer/ComparatorHelper.java:109]: Sounds good. But it is a name conversion issue, which is not related to the current issue according to your description. Can you create a separate PR to submit all your modifications about name conversions?

**Comment by bajinsheng** [comment]
> Could you show a before/after output? I noticed that the logging no longer works correctly, as newlines are currently printed as `\n`, rather than an actual linebreak.

@ZhengLin-Li  Can you also look into Manuel's comment?

**Comment by ZhengLin-Li** [comment]
> Could you show a before/after output? I noticed that the logging no longer works correctly, as newlines are currently printed as `\n`, rather than an actual linebreak.

before:

image




after:

image

File: src/sqlancer/ComparatorHelper.java [hunk 2]
```diff
@@ -87,13 +87,19 @@ public static List<String> getResultSetFirstColumnAsString(String queryString, E
...
-            state.getState().getLocalState().log(String.format("%s\n%s", firstQueryString, secondQueryString));
-            String assertionMessage = String.format("the size of the result sets mismatch (%d and %d)!\n%s\n%s",
-                    resultSet.size(), secondResultSet.size(), firstQueryString, secondQueryString);
+            String combinedQueryString = String.join(";", combinedString);
+            String secondQueryString = String.format(queryFormatString, combinedQueryString, secondResultSet.size());
+            state.getState().getLocalState()
+                    .log(String.format("%s" + System.lineSeparator() + "%s", firstQueryString, secondQueryString));
+            String assertionMessage = String.format(
+                    "the size of the result sets mismatch (%d and %d)!" + System.lineSeparator()
+                            + "FirstQuery: \"%s\", whose cardinality is: %d" + System.lineSeparator()
```
Comment (mrigger) [inline src/sqlancer/ComparatorHelper.java:99]: Since these are not variable names, I think we could just use "first query: " and "second query: "?

Comment (ZhengLin-Li) [reply src/sqlancer/ComparatorHelper.java]: make sense, done.

**Review by mrigger — APPROVED** [review]
LGTM! Based on my understanding, you addressed @bajinsheng's comments as well, so I'm merging the PR.

**Pull request merged**